source/test/TestApp/TestApp.cpp

```diff
@@ -81,7 +81,7 @@ namespace PlayFabUnit
         // Set this up for use when the tests finish
         this->clientApi = std::make_shared<PlayFab::PlayFabClientInstanceAPI>(PlayFab::PlayFabSettings::staticPlayer);
 
-#if !defined(PLAYFAB_PLATFORM_IOS) // IOS proj doesn't include PlatformLoginTest files yet
+#if !defined(PLAYFAB_PLATFORM_GDK) && !defined(PLAYFAB_PLATFORM_IOS) // IOS proj doesn't include PlatformLoginTest files yet
```

Comment:
>IOS [](start = 72, length = 3)

update this comment, is there another bug tracking fix for GDK?

Reply:
We will not be pursuing a targeted fix for GDK. I will update the comment.